### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by codrcodz**
- No changes in file content from latest config files in main bashellite repo

### CODE REVIEW

File: repos.conf.d/pypi/filter.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+bzip
+crypt
+django-server
+pwd
+setup-tools
+sickrage
+telnet
+urlib3
+urlib
+workbench
```
Comment (ericlake) [inline repos.conf.d/pypi/filter.conf:12]: We were keeping workbench and sickrage

Comment (ericlake) [reply repos.conf.d/pypi/filter.conf:12]: Actually do we even need these since we have the blacklist set in the bandersnatch config?

File: repos.conf.d/ubuntu/repo.conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+repo_url="rsync://archive.ubuntu.com/ubuntu/"
+repo_provider="rsync"
```
Comment (ericlake) [inline repos.conf.d/ubuntu/repo.conf:2]: Are we going to use apt-mirror for all deb repos?

File: repos.conf.d/fedora/provider.conf [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+linux/releases/*/Everything/source/
+linux/releases/*/Everything/x86_64/debug/
+linux/updates/testing/
+linux/updates/7/
+linux/updates/8/
+linux/updates/9/
+linux/updates/1*
+linux/updates/20
+linux/updates/21
+linux/updates/22
```
Comment (ericlake) [inline repos.conf.d/fedora/provider.conf:19]: According to https://github.com/AFCYBER-DREAM/bashellite/pull/33 lines 17 - 19 need to end in a '/'

File: repos.conf.d/fedora/provider.conf [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+linux/updates/*/SRPMS/
+linux/updates/*/x86_64/debug/
```
Comment (ericlake) [inline repos.conf.d/fedora/provider.conf:21]: Per the same PR the following should be added

+linux/updates/*/x86_64/debug/
+linux/development/
+linux/core/

File: repos.conf.d/pypi/filter.conf (hunk 1 above)
Comment (codrcodz) [reply repos.conf.d/pypi/filter.conf:12]: No. I just forgot to remove it. This file literally does nothing.

File: repos.conf.d/ubuntu/repo.conf (hunk 2 above)
Comment (codrcodz) [reply repos.conf.d/ubuntu/repo.conf:2]: Yes. I was hoping to not make any changes to the repo filter at the same time we broke out the configs from the main repo. I intentionally submitted the PR with no updates since it will be hard to follow them with no revision history.

**Comment by codrcodz** [comment]
This commit should be merged with no changes to the contents of the files so that we can have a revision history that matches the most recent commit to the main repo before we broke out the config files.
